core/tools/linux2rest/bootstrap.sh

```diff
@@ -13,6 +13,6 @@ if [[ "$(uname -m)" == "x86_64"* ]]; then
 fi
 
 # Download and install necessary libraries for linux2rest
-DEBIAN_FRONTEND=noninteractive apt --yes install libudev-dev
+DEBIAN_FRONTEND=noninteractive apt update && apt install -y --no-install-recommends libudev-dev
```

Comment:
Are you sure that we are not going to miss something important for udev ?
Did you test Linux2rest ?